interleave.py

```diff
@@ -0,0 +1,28 @@
+def interleave_function(*args: list[list[any]]) -> list[str]:
+    """
```

Comment:
I know that it is ok to take functions from other websites as long as you write that, but here its the entire exercise so I think that it is a little too much.
Im not telling you not to use functions from stackoverflow, but notice for the next time that the idea is not to take it in this scale..